2022-02-04 03:25:40,103 INFO    MainThread:18184 [wandb_setup.py:_flush():71] setting env: {}
2022-02-04 03:25:40,104 INFO    MainThread:18184 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-04 03:25:40,104 INFO    MainThread:18184 [wandb_init.py:_log_setup():371] Logging user logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_032540-nq1k8xbf\logs\debug.log
2022-02-04 03:25:40,105 INFO    MainThread:18184 [wandb_init.py:_log_setup():372] Logging internal logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_032540-nq1k8xbf\logs\debug-internal.log
2022-02-04 03:25:40,105 INFO    MainThread:18184 [wandb_init.py:init():404] calling init triggers
2022-02-04 03:25:40,106 INFO    MainThread:18184 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-04 03:25:40,106 INFO    MainThread:18184 [wandb_init.py:init():460] starting backend
2022-02-04 03:25:40,106 INFO    MainThread:18184 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-02-04 03:25:40,119 INFO    MainThread:18184 [backend.py:ensure_launched():216] starting backend process...
2022-02-04 03:25:40,227 INFO    MainThread:18184 [backend.py:ensure_launched():222] started backend process with pid: 11756
2022-02-04 03:25:40,228 INFO    MainThread:18184 [wandb_init.py:init():469] backend started and connected
2022-02-04 03:25:40,230 INFO    MainThread:18184 [wandb_init.py:init():533] updated telemetry
2022-02-04 03:25:40,330 INFO    MainThread:18184 [wandb_init.py:init():563] communicating current version
2022-02-04 03:25:42,098 INFO    MainThread:18184 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-04 03:25:42,100 INFO    MainThread:18184 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-04 03:25:42,504 INFO    MainThread:18184 [wandb_init.py:init():606] starting run threads in backend
2022-02-04 03:25:45,421 INFO    MainThread:18184 [wandb_run.py:_console_start():1810] atexit reg
2022-02-04 03:25:45,432 INFO    MainThread:18184 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-04 03:25:45,432 INFO    MainThread:18184 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-04 03:25:45,435 INFO    MainThread:18184 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-04 03:25:45,435 INFO    MainThread:18184 [wandb_init.py:init():633] run started, returning control to user process
2022-02-04 03:25:46,228 INFO    MainThread:18184 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-04 03:25:46,229 INFO    MainThread:18184 [wandb_run.py:_restore():1752] restore
2022-02-04 03:25:47,008 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 03:25:47,409 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 03:25:47,585 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7424
}

2022-02-04 03:25:47,700 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7424
}

2022-02-04 03:25:47,811 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7424
}

2022-02-04 03:25:47,922 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7424
}

2022-02-04 03:25:48,033 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7424
}

2022-02-04 03:25:48,143 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:48,255 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:48,366 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:48,478 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:48,588 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:48,697 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:49,480 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}

2022-02-04 03:25:50,289 INFO    MainThread:18184 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7424
  total_bytes: 7424
}
local_info {
}

2022-02-04 03:25:51,494 INFO    MainThread:18184 [wandb_run.py:_append_files():2180] logging synced files
